### PR TITLE
Add helper text to action menu

### PR DESCRIPTION
**Pull request by kevinp-uzi**
Added ability to add helper text in an action menu. Contents of helper text are passed to dropdown footer.

### CODE REVIEW

File: packages/ActionMenu/src/ActionMenu.tsx [hunk 1]
```diff
@@ -51,6 +51,8 @@ export interface ActionMenuProps extends React.ComponentProps<"div"> {
     renderReference: (
         props: React.HTMLProps<HTMLButtonElement>,
     ) => React.ReactElement;
+    /** Helper text to display below the options */
+    helperText?: string;
```
Comment (vicky-comeau) [inline packages/ActionMenu/src/ActionMenu.tsx:55]: I guess I'm just wondering if it is always just helper text here or should we allow the user to set whatever they want in the footer? Will this be a common thing?

File: packages/ActionMenu/src/action-menu.scss [hunk 2]
```diff
@@ -8,15 +8,25 @@
...
+  --ids-action-menu-helper-text-color: #{tokens.$grey-600};
 }
 
 [data-brand="workleap"] {
-  --ids-action-menu-font-family:  var(--hop-body-sm-font-family);
+  --ids-action-menu-font-family: var(--hop-body-sm-font-family);
   --ids-action-menu-font-size: var(--hop-body-sm-font-size);
+
+  --ids-action-menu-helper-text-size: var(--hop-body-xs-font-size);
+  --ids-action-menu-helper-text-color: var(--hop-neutral-text-weak);
```
Comment (vicky-comeau) [inline packages/ActionMenu/src/action-menu.scss:21]: We should be using Igloo's HelperText component. These properties are already set in there.

File: packages/ActionMenu/src/ActionMenu.tsx (hunk 1 above)
Comment (kevinp-uzi) [reply packages/ActionMenu/src/ActionMenu.tsx]: Checked in with a designer and indeed more versatile is better. Updated

**Review by vicky-comeau — DISMISSED** [review]

**Comment by alexasselin008** [comment]
A Chromatic test should be added for this new feature

**Review by alexasselin008 — COMMENTED** [review]
A Chromatic test should be added for this new feature in ActionMenu.stories.tsx

**Review by fraincs — DISMISSED** [review]

**Review by alexasselin008 — CHANGES_REQUESTED** [review]
A Chromatic test should be added for this new feature in ActionMenu.stories.tsx

**Comment by vicky-comeau** [comment]
Add "@igloo-ui/helper-text": "^0.2.16" to devDependencies in ActionMenu package.json and then do `yarn install`

**Review by vicky-comeau — APPROVED** [review]

**Review by vicky-comeau — APPROVED** [review]

**Review by alexasselin008 — APPROVED** [review]

**Pull request merged**